conference/cart.py

```diff
@@ -267,6 +266,17 @@ def send_order_confirmation_email(order: Order, current_site) -> None:
     )
 
 
+def is_early_bird_sold_out():
+    eb_ticket_orders = Ticket.objects.filter(
+        fare__conference=settings.CONFERENCE_CONFERENCE,
+        frozen=False,
+        # orderitem__order___complete=True,
+        fare__code__regex=FARE_CODE_REGEXES["types"][FARE_CODE_TYPES.EARLY_BIRD]
+    )
+
+    return eb_ticket_orders.count() > settings.EARLY_BIRD_ORDER_LIMIT
```

Comment:
`settings.EARLY_BIRD_ORDER_LIMIT` ? I thought this was supposed to be configurable via Fares.